Python/sysmodule.c

```diff
@@ -2232,6 +2232,11 @@ make_impl_info(PyObject *version_info)
     if (res < 0)
         goto error;
 
+    res = PyDict_SetItemString(impl_info, "opt_levels",
+                               Py_BuildValue("(sii)", "", 1, 2));
```

Comment:
The ``Py_BuildValue()`` call needs to be handled like the ``PyLong_FromLong()`` call just above. The current code causes a reference leak (or a crash if the ``Py_BuildValue()`` call fails).

Reply:
Thank you!  I've pushed the change.